### PR TITLE
Add responsive desktop version

### PR DESCRIPTION
**Pull request by rbanos-mv**
Add responsive classes and modifications to the html for the desktop version of the portfolio.

### CODE REVIEW

File: css/about.css [hunk 1]
```diff
@@ -122,3 +133,35 @@
...
   color: #36b37e;
 }
+
+@media (min-width: 768px) {
+  .about {
+    max-width: 1175px;
+    background-image: url("../images/illustration-about-me-desktop.svg");
+    background-position: top center;
+    background-repeat: no-repeat;
+  }
```
Comment (keroloussamy) [inline css/about.css:143]: - [ ] Kindly display the whole photos here to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: css/about.css [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
-.about-container {
+.about-header {
   display: flex;
   flex-direction: column;
-  margin-top: 35px;
-  padding: 0;
+  align-items: center;
+  align-self: center;
+  padding: 24px;
+  max-width: 920px;
```
Comment (keroloussamy) [inline css/about.css:23]: - [ ] Kindly on your mobile version add these two photos to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: css/styles.css [hunk 3]
```diff
@@ -170,6 +170,49 @@ h2 {
...
+    width: 100%;
+    height: 72px;
+  }
+
+  .wrapper {
+    max-width: 1440px;
+    background-image: url("../images/header-illustration-desktop.svg");
+    background-position: top center;
+    background-repeat: no-repeat;
+  }
```
Comment (keroloussamy) [inline css/styles.css:194]: - [ ] Kindly make the background here not go to the `works` section too, make it for the `headline` section only, to match the Figma design.

Your design
image

Figma design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @rbanos-mv, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/about.css (hunk 1 above)
Comment (rbanos-mv) [reply css/about.css]: Done.
image

File: css/about.css (hunk 2 above)
Comment (rbanos-mv) [reply css/about.css:30]: Done.
image

File: css/styles.css (hunk 3 above)
Comment (rbanos-mv) [reply css/styles.css:195]: Done.
image

**Review by smunozmo — APPROVED** [review]
Hi @rbanos-mv, 👋 

Great job making all the required changes, your project is looking amazing! There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**